todoapp/users/serializers.py

```
@@ -1,7 +1,7 @@
 from django.contrib.auth import authenticate
 from django.contrib.auth import get_user_model
 from django.contrib.auth.hashers import make_password
-from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import gettext_lazy as _
```

Ruff (0.12.2)

`django.utils.translation.gettext_lazy` imported but unused

Remove unused import: `django.utils.translation.gettext_lazy`

(F401)